src/test/scala/io/github/pauljamescleary/petstore/Arbitraries.scala

```diff
@@ -14,6 +15,9 @@ import io.github.pauljamescleary.petstore.domain.users._
 
 trait PetStoreArbitraries {
 
+  val userNameLength = 16
+  val userNameGen: Gen[String] = Gen.listOfN(userNameLength, Gen.alphaChar).map(_.mkString)
```

Comment:
I took the liberty of creating a username generator because the specs were failing for too large username or usernames containing some Asian characters(probably UTF-16 encoding). 

I kept things simple, 16 alpha characters, since this is only for test purposes but, let me know if you'd like to consider more complex scenarios for the username generator